src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -89,6 +94,11 @@ public override void Validate()
         /// </summary>
         public PathString CallbackPath { get; set; }
 
+        /// <summary>
+        /// Remote protocol errors reported to the CallbackPath will be redirected to this path. The default value is "/Home/Error".
+        /// </summary>
+        public PathString RemoteFailureRedirect { get; set; } = "/Home/Error";
```

Comment:
Nit, does this need some kind of Uri/Path suffix to indicate that its the RedirectPath/Uri?

Reply:
RemoteFailurePath?